examples/catalog/bin/sample_page.dart

```diff
@@ -255,8 +291,8 @@ void main(List<String> args) {
   } catch (error) {
     logError(
       'Error: sample_page.dart failed: $error\n'
-      'This sample_page.dart app expects to be run from the examples/catalog directory. '
-      'More information can be found in examples/catalog/README.md.'
+          'This sample_page.dart app expects to be run from the examples/catalog directory. '
+          'More information can be found in examples/catalog/README.md.',
```

Comment:
This is definitely less clear than the original.

Reply:
there are bugs filed on the formatter tracking this issue. I expect this is one case that can be fixed.